src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -3202,7 +3202,11 @@ namespace winrt::TerminalApp::implementation
             // elevated version of the Terminal with that profile... that's a
             // recipe for disaster. We won't ever open up a tab in this window.
             newTerminalArgs.Elevate(false);
-            _CreateNewTabFromPane(_MakePane(newTerminalArgs, false, connection));
+            const auto newPane = _MakePane(newTerminalArgs, false, connection);
+            newPane->WalkTree([](auto pane) {
```

Comment:
Not sure if there is any way currently for a defterm invocation to open up a tree of panes, but for future-proofing purposes we call `WalkTree` here